docs/experimental_grpc_python_api.rst

```diff
@@ -35,13 +35,13 @@ This module contains most of the core functionality of the library
 including setting up a connection, sending requests and receiving
 response to/from an active Triton server.
 
-.. automodule:: tritongrpcclient.core
+.. automodule:: tritongrpcclient.grpcclient
    :members:
 
 Client Utils
 ------------
 
 This module exposes additional supporting utilities.
 
-.. automodule:: tritongrpcclient.utils
+.. automodule:: tritonclientutils.utils
```

Comment:
This is also a little redundant but I guess ok since there are other things in tritonclientutils other than the actual utils